containers/python/NLTK/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.4"
+
+services:
+  grpc-api:
+    image: ghcr.io/deephaven/nltk-base:${VERSION:-latest}
```

Comment:
This is not the correct image, which makes me think this compose file hasn't been tested.